crt_portal/cts_forms/admin.py

```diff
@@ -32,7 +32,7 @@
                      ProtectedClass, Report, ResponseTemplate, DoNotEmail,
                      JudicialDistrict, RetentionSchedule, RoutingSection, RoutingStepOneContact, Tag,
                      VotingMode, Campaign, ReferralContact, BannerMessage, SavedSearch, NotificationPreference)
-from .signals import get_client_ip
+from crt_portal.utils.request_utils import get_client_ip
```

Comment:
You can drop the `crt_portal.`

```suggestion
from utils.request_utils import get_client_ip
```